src/app-layer-smtp.c

```diff
@@ -147,6 +147,8 @@ SCEnumCharMap smtp_decoder_event_table[ ] = {
       SMTP_DECODER_EVENT_MIME_LONG_HEADER_VALUE },
     { "MIME_LONG_BOUNDARY",
       SMTP_DECODER_EVENT_MIME_BOUNDARY_TOO_LONG },
+    { "MIME_LONG_FILENAME",
```

Comment:
Should not we add a rule for this event in rules/smtp-event.rules ?

Reply:
Thanks for pointing this out.

We should -- I'll make an update.